packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -383,6 +395,12 @@ class _DragTargetState<T> extends State<DragTarget<T>> {
   final List<_DragAvatar<T>> _candidateAvatars = <_DragAvatar<T>>[];
   final List<_DragAvatar<dynamic>> _rejectedAvatars = <_DragAvatar<dynamic>>[];
 
+  void onDrag(_DragAvatar<dynamic> avatar, Offset lastOffset) {
+    if (widget.onDrag != null) {
```

Comment:
Done. (Why not follow https://www.dartlang.org/guides/language/effective-dart/style#do-use-curly-braces-for-all-flow-control-structures?)